VogtPayroll2/PayrollManager.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+
+
+namespace VogtPayroll2
+{
+    class PayrollManager
+    {
+
+        #region GetNetPayOfAllEmployees
+        /// <summary>
+        /// Sorts the netpay of all employees
+        /// </summary>
+        /// <param name="employees">Employees passed in by a list</param>
+        /// <returns>A sorted list</returns>
+        public List<Employee> GetNetPayOfAllEmployees(List<Employee> employees)
+        {
+            List<Employee> sortedList = new List<Employee>();
```

Comment:
don't initialize a new list right before you overwrite it! its creating garbage (wasting memory) that the application has to discard.